src/opendr/engine/target.py

```diff
@@ -45,6 +45,60 @@ def __init__(self):
         self.confidence = None
         self.action = None
 
+    @property
+    def data(self):
+        """
+        Getter of data field.
+        This returns the internal representation of the data.
+        :return: the actual data held by the object
+        :rtype: Type of data
+        """
+        return self._data
+
+    @data.setter
+    def data(self, data):
+        """
+        Setter for data. This will perform the necessary type checking (if needed).
+        :param: data to be assigned to the object
+        """
+        self._data = data
+
+    @property
+    def confidence(self):
+        """
+        Getter of confidence field.
+        This returns the confidence for the current target.
+        :return: the confidence held by the object
+        :rtype: Type of data
+        """
+        return self._confidence
+
+    @confidence.setter
+    def confidence(self, confidence):
+        """
+        Setter for the confidence field. This will perform the necessary type checking (if needed).
+        :param: confidence to be used for assigning confidence to this object
+        """
+        self._confidence = confidence
+
+    @property
+    def action(self):
+        """
+        Getter of action field.
+        This returns the selected/expected action.
+        :return: the action data held by the object
+        :rtype: Type of data
```

Comment:
Fixed. Please note that the Action class does not yet exists (even though it is specified). It will be added in a future version of the toolkit (when active perception algorithms will be provided).